go.sum

```diff
@@ -18,6 +18,7 @@ code.cloudfoundry.org/credhub-cli v0.0.0-20190415201820-e3951663d25c h1:qdDzK71H
 code.cloudfoundry.org/credhub-cli v0.0.0-20190415201820-e3951663d25c/go.mod h1:oZAAU/4R7ssJ+kighWL4VLyyyY+Utm8H9r721AE3edQ=
 code.cloudfoundry.org/garden v0.0.0-20181108172608-62470dc86365 h1:W9nK4TUFQCzLniQN2xykFvzAteV0FogE94bVj8Ulu8A=
 code.cloudfoundry.org/garden v0.0.0-20181108172608-62470dc86365/go.mod h1:9edl55ou1627lC7chvtltZ561+iITkNTzkZsLgohU8s=
+code.cloudfoundry.org/lager v1.1.0 h1:v0RELJ2jqTeF2DW7PNjZaaGlrXbVxJBVz3uLxdP3fuY=
```

Comment:
hmm, this is interesting

Reply:
fyi: just rebased and now it only changes baggageclaim lines